WordPress/Classes/ViewRelated/System/Floating Create Button/CreateButtonCoordinator.swift

```diff
@@ -141,7 +150,10 @@ import WordPressFlux
     }
 
     @objc func showCreateButton() {
-        noticeContainerView = noticeAnimator.present(notice: notice, in: viewController!.view, sourceView: button)
+        shownTooltipCount += 1
```

Comment:
incrementing before presenting means that we will see the button `maximumTooltipView - 1` times, unless we change the condition in `shownTooltipCount` to `>`.